lifecycle-operator/controllers/common/config/config.go

```diff
@@ -45,3 +48,11 @@ func (o *ControllerConfig) SetCloudEventsEndpoint(endpoint string) {
 func (o *ControllerConfig) GetCloudEventsEndpoint() string {
 	return o.cloudEventsEndpoint
 }
+
```

Comment:
let's add a unit test for these now functions to make codecov happy :)

Reply:
sure 😭